model-optimizer/extensions/ops/split.py

```diff
@@ -44,7 +44,7 @@ def infer(node):
 
         axis = node.in_port(1).data.get_value() if op == 'VariadicSplit' else node.soft_get('axis', None)
         assert axis is not None, '{} `axis` is unknown for node {}'.format(op, name)
-        assert axis.ndim == 0, '{} `axis` should be scalar, but it`s not for node {}'.format(op, name)
+        assert axis.ndim == 0 or len(axis) == 1 and axis.ndim == 1, '{} `axis` should be scalar, but it`s not for node {}'.format(op, name)
```

Comment:
Update the comment. Now the input could be a scalar or a tensor with shape `[1]`

Reply:
Done.